docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
+
+![Settings Menu](images/settings-menu.png)
+
+The left pane contains several sections to view different pieces of data about Polynote. 
+
+### About 
+The about menu contains information about your instance's version and most recent server commit. 
+
+### Hotkeys 
+The hotkeys menu contains a list of all supported keyboard shortcuts in Polynote. 
+
+### Preferences 
+The preferences menu contains two sections - _UI preferences_ and _storage_.
+
+#### UI Preferences 
+The preferences menu is where you can customize the following settings: 
+
+- **vim**: Whether VIM input mode is enabled for code cells 
+- **theme**: Light or Dark application color scheme 
+- **notifications**: Whether Polynote can send you browser notifications. 
+
+#### Storage 
+The storage menu displays key-value pairs of all the data Polynote it storing in your browser's local storage, 
+and provides easy access to wipe that data. 
+
+### Open Kernels 
+The open kernels menu contains a list of all existing kernels for existing notebooks. You can view their path, 
+status (idle, running, or dead), and start/kill/jump to all of these kernels. 
+
+### Client-side Backups 
```

Comment:
I would add: 

1. It may take a long time to load this page if you have many backups
2. Unfortunately there's no way to automatically recover these backups - a user trying to recover a backup would need to manually parse the JSON (which is not in ipynb format).